src/bpforc.h

```diff
@@ -55,6 +55,11 @@ class MemoryManager : public SectionMemoryManager
                                StringRef SectionName,
                                bool isReadOnly) override;
 
+  ~MemoryManager()
+  {
+    deregisterEHFrames();
```

Comment:
I'm in awe of how you figured this out

Reply:
debuginfod on debian saved me here. Once i was certain that it was valid at some point I just set a watch point on the internal (libgcc) variable that holds them and have the backtrace tell it all. Getting the required symbols in the bionic based builds was impossible

Reply:
Amazing work! I'm gonna have to read up on the tools you used.